frontend/packages/patternfly/src/components/notification-drawer/notification-drawer-heading.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+const NotificationDrawerHeading: React.FC<NotificationDrawerHeadingProps> = ({
+  count,
+  children,
+}) => (
+  <div className="pf-c-notification-drawer">
+    <div className="pf-c-notification-drawer__header">
+      <h1 className="pf-c-notification-drawer__header-title">Notifications</h1>
+      <span className="pf-c-notification-drawer__header-status">{`${count} unread`}</span>
```

Comment:
Not quite sure how to toggle `read/unread`.  Clicking messages currently does a navigation & closes the drawer.  If I am on `http://localhost:9000/monitoring/alerts/<id>?alertname=<otherID>` then it will repeatedly close the drawer. Pondering this, it might be nice if it stayed open as I have to keep opening the drawer to cycle through the alerting types, though I could also just click back into the main `Alerting` page.

Reply:
@benjaminapetersen read/unread doesn't exist for openshift. the openshift alerts don't have the read/unread state, so we are using (according to @cshinn 's design: https://github.com/openshift/openshift-origin-design/pull/306), read state for the section header counts, and unread state for notification items in the drawer.

Reply:
I'm good with that, but it does imply that clicking the messages would toggle state:

![Screen Shot 2020-01-23 at 2 59 02 PM](https://user-images.githubusercontent.com/280512/73019130-ffb94a00-3df0-11ea-9d68-2402a7f6b402.png)

Since we can't actually click to toggle, maybe we should use word `total` instead of `read/unread`?